src/app/download/download.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import {TestBed} from '@angular/core/testing';
+
+import {DownloadService} from './download.service';
+
+describe('DownloadService', () => {
```

Comment:
Add test cases for generatedHtml e.g.

- should contain defined landingUrl element
- should contain defined callToAction enum
- etc.

Reply:
Completed this item in download tests PR